backend/pkg/httpserver/list_missing_one_implementation_counts.go

```diff
@@ -23,6 +23,21 @@ import (
 	"github.com/GoogleChrome/webstatus.dev/lib/gen/openapi/backend"
 )
 
+var ErrNoMatchingMobileBrowser = errors.New("browser does not have a matching mobile browser")
+
+func getDesktopsMobileProduct(browser backend.BrowserPathParam) (backend.BrowserPathParam, error) {
+	switch browser {
```

Comment:
Related to this [comment](https://github.com/GoogleChrome/webstatus.dev/pull/1460#discussion_r2073873048) here about using default.